### PR TITLE
💌 Add notification example to our docs

### PR DESCRIPTION
**Pull request by mj426382**

### CODE REVIEW

File: packages/docs/src/examples/Notifications.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            <button onClick={() => wrapEther()}>Wrap ether</button>
+            <p>Status: {status}</p>
+            <p>Notifications</p>
+            {nots.length !== 0 && 
+            (<table>
+                <th>Id</th>
+                <th>Type</th>
+                <th>Date</th>
+            {nots.map((notification) => {
+                console.log(notification)
```
Comment (rzadp) [inline packages/docs/src/examples/Notifications.tsx:53]:
```suggestion
```

File: packages/docs/src/examples/Notifications.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      <App />
+    </DAppProvider>,
+    document.getElementById('root')
+)
+
+export function App() {
+    const { notifications } = useNotifications()
+    const { account, chainId, activateBrowserWallet } = useEthers()
+    const isSupportedChain = SUPPORTED_TEST_CHAINS.includes(chainId)
+    const [nots, setNots] = useState([])
```
Comment (rzadp) [inline packages/docs/src/examples/Notifications.tsx:20]: Weird shorthand

```suggestion
    const [notifications, setNotifications] = useState([])
```

File: packages/docs/src/examples/Notifications.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+export function App() {
+    const { notifications } = useNotifications()
+    const { account, chainId, activateBrowserWallet } = useEthers()
+    const isSupportedChain = SUPPORTED_TEST_CHAINS.includes(chainId)
+    const [allNotifications, setAllNotifications] = useState([])
+
+    useEffect(() => {
+        const mergedArray = allNotifications.concat(notifications)
+        const filteredArray = mergedArray.filter(function(el , id){
```
Comment (rzadp) [inline packages/docs/src/examples/Notifications.tsx:24]:
```suggestion
        const uniqueNotifications = mergedArray.filter(function(el , id){
```

**Review by rzadp — CHANGES_REQUESTED** [review]
@mj426382 Can you set configuration `expirationPeriod` to 0, and implement `useNotifications` so that the notifications do not expire? Then we don't need to track `allNotifications` in the example.

File: packages/docs/docs/guides/transactions.mdx [hunk 4]
```diff
@@ -182,3 +182,13 @@ For example :
 ``notificationContent`` is an object that holds information about what title and icon to show.
 You have to remember that object in ``notifications`` array may not contain transaction field
  (that's why there is if statement).
+
+**Live example**
+
+App will deposit 1 wei to Wrapped Ether contract.
```
Comment (rzadp) [inline packages/docs/docs/guides/transactions.mdx:188]:
```suggestion
App will deposit 1 wei to Wrapped Ether contract and display transaction notification.
```

This is the main thing of the example, should be said upfront.

File: packages/docs/docs/guides/transactions.mdx [hunk 5]
```diff
@@ -182,3 +182,13 @@ For example :
...
+Connect a MetaMask wallet **and switch to a test network**, such as Kovan or Ropsten.
+The app will also display transaction notifications.
```
Comment (rzadp) [inline packages/docs/docs/guides/transactions.mdx:190]:
```suggestion
```

**Review by rzadp — CHANGES_REQUESTED** [review]

**Comment by rzadp** [comment]
Screenshot 2022-03-28 at 10 23 32

This is `constants/Weth.ts` file right? Should be titled, like `App.tsx` is.

**Comment by rzadp** [comment]
Please remove `id` from table

**Review by rzadp — APPROVED** [review]

**Review by dmaretskyi — APPROVED** [review]

**Comment by KevinsBobo** [comment]
https://github.com/TrueFiEng/useDApp/commit/751327e7f84ec1a947deb2689467f85f9bc65f1a#r70259189
This change introduces new problems:
https://github.com/TrueFiEng/useDApp/issues/640

**Comment by rzadp** [comment]
> [751327e#r70259189](https://github.com/TrueFiEng/useDApp/commit/751327e7f84ec1a947deb2689467f85f9bc65f1a#r70259189) This change introduces new problems: #640

Addressed by https://github.com/TrueFiEng/useDApp/pull/655

**Pull request merged**